exporters/exporter.go

```diff
@@ -16,13 +16,13 @@ type ExportData interface {
 // BlockExportData is provided to the Exporter on each round.
 type BlockExportData struct {
```

Comment:
Is this going to live in a "common" folder?

Reply:
It should, I can move it to one.